lib/install/phases/pull.go

```diff
@@ -165,10 +165,15 @@ func (p *pullExecutor) applyPackageLabels() error {
 		constants.TeleportPackage,
 		constants.GravityPackage,
 	}
+	purposeLabels := []string{
+		pack.PurposePlanetConfig,
+		pack.PurposePlanetSecrets,
+	}
 	var locators []loc.Locator
-	err := pack.ForeachPackageInRepo(p.LocalPackages, defaults.SystemAccountOrg,
+	err := pack.ForeachPackage(p.LocalPackages,
```

Comment:
This is to be able to update runtime (planet) configuration and secret package labels as these are not in the default repository but in cluster's.